package.json

```diff
@@ -59,8 +59,9 @@
     "mkdirp": "^0.5.1",
     "moment": "^2.22.1",
     "node-fetch": "^2.6.0",
+    "openfin-service-config": "^1.0.2",
```

Comment:
Why this addition? Or why did we not need this before?

Reply:
We made the service config module a dev dependency in the tooling. I have updated tooling again to add it back as a normal dependency.  The overall problem is that the tooling/spawn utils imports a type from the config so it is needed, even if it is not being used.